components/dropbox/actions/upload-file/upload-file.mjs

```diff
@@ -27,17 +25,10 @@ export default {
       label: "File Name",
       description: "The name of your new file (make sure to include the file extension).",
     },
-    fileUrl: {
+    file: {
```

Comment:
This needs a major version bump since we're removing existing props and adding a required one (may break the user workflow)

Also, optionally as a standard I've been keeping `filePath` as the prop name (to maintain existing user input there) instead of changing it to `file`